pkg/mockclient/client.go

```diff
@@ -91,11 +93,18 @@ func (c *Client) callMock(mockAPI, mockReqBody string) {
 	}
 	mockRes, err := hc.Do(mockReq)
 	if err != nil {
-		require.NoError(c.T, err, "Failed to send request to mock server.")
+		require.NoError(c.T, err, "Failed to send request to MockServer.")
 	}
-	// MockServer verification returns 202 on success and 406 on failure
-	if mockRes.StatusCode != http.StatusAccepted {
-		require.NoError(c.T, err,
-			fmt.Sprintf("Mock server verification did not meet expectations and failed with status: %s", mockRes.Status))
+
+	// all went well so return (clears & /reset return 200 whilst /verify & /expectation return 201)
+	if mockRes.StatusCode >= 200 && mockRes.StatusCode <= 299 {
+		return
+	}
+
+	// something went wrong so return the error message (Note: MockServer /verify returns 406 on failure)
+	b, err := ioutil.ReadAll(mockRes.Body)
+	if err != nil {
+		assert.Fail(c.T, fmt.Sprintf("MockServer call failed with status: %s", mockRes.Status))
 	}
+	assert.Fail(c.T, fmt.Sprintf("MockServer call to /%s failed with status: %s. Error message from MockServer is: %s", mockAPI, mockRes.Status, string(b)))
```

Comment:
Does this error message look nice with last string(b)? Do you get like a nice json that is easy to read?

Reply:
Hmm good point. It looks ok I'd say but i'll see if I can make it more beautiful.

Reply:
<img width="1332" alt="Screen Shot 2019-04-04 at 15 16 52" src="https://user-images.githubusercontent.com/26872356/55558669-d659f100-56ec-11e9-895a-8e35300f671a.png">

Actually it looks pretty ok.

Reply:
Nice! :D Was wondering since some json outputs look like a dumpster fire

Reply:
This is a backquote escaped, reasonably indented, small dumpster fire :)